app/assets/javascripts/govuk_publishing_components/lib/header-navigation.js

```diff
@@ -51,12 +51,12 @@
         if (targetClass.indexOf('js-visible') !== -1) {
           target.setAttribute('class', targetClass.replace(/(^|\s)js-visible(\s|$)/, ''))
           if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
-            window.GOVUK.analytics.trackEvent('headerClicked', 'menuClosed', { label: 'none' })
```

Comment:
This is sadly going to trigger for menu buttons for core layout applications: code is [here](https://github.com/alphagov/static/blob/main/app/views/root/proposition_menu.html.erb#L3). Example of a page using this is [here](https://www.gov.uk/government/how-government-works).

Are we happy with this as a possible bit of chuff?

Reply:
Right, well that's fun. I've updated this to read from a `data-name` attribute so it shouldn't confuse between the two, and there's a fallback in case there's no `data-name` present on the button.